src/app/Components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  export let isSelected: boolean = name === selectedLanguage;
+  export let className: string = "avatar";
+  if (isSelected) {
+    className = className + " selected";
+  }
+
+  /**
+   * Handles the click when a language is selected from the sidebar.
+   * Should change the selected language variable and change content in the Languages component.
+   * TODO: Replace stub
+   *
+   * @param {string} languageName - A the name of the language selected.
+   * @return {void}
+   */
+  const handleClick = (languageName: string): void => {
+    console.log(languageName + " button selected");
+  };
+</script>
+
+<style>
+  .avatar {
+    width: 50px;
+    height: 50px;
+    display: flex;
+    background-color: #fff;
+    align-items: center;
+    justify-content: center;
+    border-radius: 100%;
+    margin-top: 40px;
+    font-family: Cabin, sans-serif;
```

Comment:
**[Suggestion]**: font-family has appeared quite a few times, can we use it in global stying